.appveyor.yml

```
@@ -12,54 +11,52 @@ branches:
 
 environment:
     matrix:
-        - PHP_VERSION: 7.1-Win32-VC14
+        - PHP_VERSION: 7.1
           DEPENDENCIES: lowest
-        - PHP_VERSION: 7.1-Win32-VC14
+        - PHP_VERSION: 7.1
           DEPENDENCIES: highest
-        - PHP_VERSION: 7.2-Win32-VC15
+        - PHP_VERSION: 7.2
           DEPENDENCIES: lowest
-        - PHP_VERSION: 7.2-Win32-VC15
+        - PHP_VERSION: 7.2
           DEPENDENCIES: highest
-        - PHP_VERSION: 7.3-Win32-VC15
+        - PHP_VERSION: 7.3
           DEPENDENCIES: lowest
-        - PHP_VERSION: 7.3-Win32-VC15
+        - PHP_VERSION: 7.3
           DEPENDENCIES: highest
-        - PHP_VERSION: 7.4-Win32-VC15
+        - PHP_VERSION: 7.4
           DEPENDENCIES: lowest
-        - PHP_VERSION: 7.4-Win32-VC15
+        - PHP_VERSION: 7.4
           DEPENDENCIES: highest
 
 matrix:
     fast_finish: true
 
 cache:
     - composer.phar
-    - '%LocalAppData%\Composer\files'
-    - 'c:\php -> .appveyor.yml'
+    - '%LOCALAPPDATA%\Composer\files -> composer.lock'
```

IMHO we shouldn't thrash this cache completely only because the .lock changes. This is only Composer's cache, so we can normally partially reuse it. It would be updated at the end of the build anyway.
```suggestion
    - '%LOCALAPPDATA%\Composer\files'
```

---

Also, are we caching `composer.phar`?

---

>we shouldn't thrash this cache completely only because the .lock changes

We don't commit the `composer.lock` file, so the cache would not be wiped out anyway, thus I reverted the change

>Also, are we caching `composer.phar`?

We are, but it never worked because the path is wrong. By the way, it's a really small file so imho it makes sense to always download it to be sure we always get the latest version

---

I suggest you to use `composer self-update` when the PHAR is present.